### DISCUSSION TITLE
OBS Now Playing html

### DISCUSSION THREAD

**Discussion by jessikatme** [pear-devs/pear-desktop#1406] [Ideas]
Hi!

For YTMDesktop someone do a HTML file to get song artist and cover image to use as Now Playing browser source in OBS, here the link
https://gist.github.com/Twanislas

The way to use it is with the activating the Remote Control in YTMDesktop, that option is not available in your youtube-music, I'm not a developer, for that reason I add it as an idea.

Best regards.

**Comment by JellyBrick** [comment]
1. Enable tuna-obs plugin in YTM
2. Install tuna plugin to OBS (https://github.com/univrsal/tuna)
